### PR TITLE
Feature/4790 data collection extended

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
The following data is collected (for details see tech spec)
Exposure Windows (from the Risk Calculation) that ENF has newly identified since the previous submission of usage data
Registered test and test result (if available)
Key Submission State, which describes if the client has submitted keys or could do so
Client Metadata

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4790

## Screenshots
N/A

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Models/Metadata/Client Metadata/ClientMetadata.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+struct ClientMetadata: Codable {
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Models/Metadata/Client Metadata/ClientMetadata.swift:7]: missing MARKs

File: src/xcode/ENA/ENA/Source/Models/Metadata/Key Submission Metadata/KeySubmissionMetadata.swift [hunk 2]
```diff
@@ -0,0 +1,75 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct KeySubmissionMetadata: Codable {
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Models/Metadata/Key Submission Metadata/KeySubmissionMetadata.swift:7]: missing MARKs

File: src/xcode/ENA/ENA/Source/Models/Metadata/Key Submission Metadata/KeySubmissionMetadataService.swift [hunk 3]
```diff
@@ -0,0 +1,90 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+class KeySubmissionService {
+	private var secureStore: Store
```
Comment (KaiTeuber) [inline src/xcode/ENA/ENA/Source/Models/Metadata/Key Submission Metadata/KeySubmissionMetadataService.swift:8]: move to private section please

**Review by pascal-brause — APPROVED** [review]

**Review by KaiTeuber — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Models/Metadata/Client Metadata/ClientMetadata.swift (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Models/Metadata/Client Metadata/ClientMetadata.swift:7]: Fixed

File: src/xcode/ENA/ENA/Source/Models/Metadata/Key Submission Metadata/KeySubmissionMetadataService.swift (hunk 3 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Models/Metadata/Key Submission Metadata/KeySubmissionMetadataService.swift]: Fixed

File: src/xcode/ENA/ENA/Source/Models/Metadata/Key Submission Metadata/KeySubmissionMetadata.swift (hunk 2 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Models/Metadata/Key Submission Metadata/KeySubmissionMetadata.swift:7]: Fixed

**Pull request merged**